### DISCUSSION TITLE
[Script request] immich

### DISCUSSION THREAD

**Discussion by vittory003** [community-scripts/ProxmoxVE#43] [Request script] [answered]
### Application Name

Immich

### Website

https://github.com/immich-app/immich

### Description

Self-hosted photo and video management solution

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Zedarius5** [comment, marked as answer]
Check out https://community-scripts.github.io/Proxmox/scripts?id=Dockge - it has an option to install Immich during its installation. I installed it this way and it works great.

**Reply by vittory003** [reply]
Thank you, I didn't notice it. Bye

**Comment by havardthom** [comment]
For anyone wanting an Immich LXC, this is the discussion you should bump: https://github.com/immich-app/immich/discussions/1657

**Reply by MickLesk** [reply]
I spoke to Alex (the founder of the project). He said some time ago that too much is changing to deploy it natively. I am partly involved in the project, mainly for smaller issues and translations. Microservices will be abolished at some point, etc.

The original statement is that there won't be a real release until then. Probably 2.0.0

**Reply by oerix** [reply]
> I spoke to Alex (the founder of the project). He said some time ago that too much is changing to deploy it natively. I am partly involved in the project, mainly for smaller issues and translations. Microservices will be abolished at some point, etc.
> 
> The original statement is that there won't be a real release until then. Probably 2.0.0

Pardon my ignorance, but maybe it could be based on a Docker-less builds like these scripts? 

https://github.com/loeeeee/immich-in-lxc
https://github.com/arter97/immich-native

**Reply by MickLesk** [reply]
Doesn't change the fact that too much will change in the future, e.g. redis will be dropped. I don't want to have a script where I might have to rework every version change. Everyone can feel free to use the repos and set it up on the basis of a Debian or Ubuntu LXC.

**Comment by gabriellxxix** [comment]
This project details how to implement Immich on an LXC, VM, or even a bare-metal installation. It seems to work well—creating a script for automation would be a great idea.


https://github.com/loeeeee/immich-in-lxc
